src/main/java/roomescape/db/ReservationDao.java

```diff
@@ -0,0 +1,65 @@
+package roomescape.db;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import javax.sql.DataSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.BeanPropertySqlParameterSource;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+
+
+@Repository
+public class ReservationDao {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    private final SimpleJdbcInsert jdbcInsert;
```

Comment:
```suggestion
public class ReservationDao {
    private final JdbcTemplate jdbcTemplate;
    private final SimpleJdbcInsert jdbcInsert;
```
컨벤션 부분들도 다른 부분과 통일해보면 좋을것 같아요!